### ISSUE TITLE
`cordova-andoid 7` unsupported?

### ISSUE THREAD

**Issue by le93us** [ratson/cordova-plugin-admob-free#168]
```sh
cordova start demo
killall java
cordova platform add android
cordova build android
```

... and I get:
```sh
(node:21889) UnhandledPromiseRejectionWarning: Error: /home/lex/Desktop/myCordovaApp/platforms/android/gradlew: Command failed with exit code 1 Error output:
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:27: error: package name.ratson.cordova.admob.banner does not exist
import name.ratson.cordova.admob.banner.BannerExecutor;
                                       ^
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:28: error: package name.ratson.cordova.admob.interstitial does not exist
import name.ratson.cordova.admob.interstitial.InterstitialExecutor;
                                             ^
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:29: error: package name.ratson.cordova.admob.rewardvideo does not exist
import name.ratson.cordova.admob.rewardvideo.RewardVideoExecutor;
                                            ^
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:44: error: cannot find symbol
    private BannerExecutor bannerExecutor = null;
            ^
  symbol:   class BannerExecutor
  location: class AdMob
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:45: error: cannot find symbol
    private InterstitialExecutor interstitialExecutor = null;
            ^
  symbol:   class InterstitialExecutor
  location: class AdMob
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:46: error: cannot find symbol
    private RewardVideoExecutor rewardVideoExecutor = null;
            ^
  symbol:   class RewardVideoExecutor
  location: class AdMob
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:70: error: cannot find symbol
            bannerExecutor = new BannerExecutor(this);
                                 ^
  symbol:   class BannerExecutor
  location: class AdMob
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:73: error: cannot find symbol
            interstitialExecutor = new InterstitialExecutor(this);
                                       ^
  symbol:   class InterstitialExecutor
  location: class AdMob
/home/lex/Desktop/myCordovaApp/platforms/android/app/src/main/java/name/ratson/cordova/admob/AdMob.java:76: error: cannot find symbol
            rewardVideoExecutor = new RewardVideoExecutor(this);
                                      ^
  symbol:   class RewardVideoExecutor
  location: class AdMob
9 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 13s
```

I get `cordova-android` v7.0, in your basic example - `6.4.0`.

As I understand, android 7 and higher are not supported?

Or this problem can be fixed?

**Comment by liazbd** [comment]
same problem, looking for solution ..

**Comment by X0t0** [comment]
same problem here

**Comment by fthkrtl35** [comment]
for now

> platforms/android/src/name/ratson/cordova/admob/banner platforms/android/src/name/ratson/cordova/admob/interstitial platforms/android/src/name/ratson/cordova/admob/rewardvideo

copy to 

> platforms/android/app/src/main/java/name/ratson/cordova/admob/

**Comment by X0t0** [comment]
@fthkrtl35  i did what wrote but i'm having this error now : 
`Dex: Error converting bytecode to dex:
Cause: PARSE ERROR:
class name (com/google/android/gms/R) does not match path (Com/google/android/gms/R.class)
...while parsing Com/google/android/gms/R.class`

**Comment by fthkrtl35** [comment]
@X0t0 i think you take an error from another plugin

**Comment by X0t0** [comment]
@fthkrtl35  i think it's from admob sdk that installed with this plugin

**Comment by srikanth-wgl** [comment]
Can you guys please check this issue once using the latest version of the plugin (v0.15.0). This should be fixed now.

**Comment by ivanov84** [comment]
@srikanth-wgl I had error with admb-free@0.15.0:

BUILD FAILED

Total time: 19.662 secs
* What went wrong:
Execution failed for task ':transformClassesWithDexForArmv7Release'.
> com.android.build.api.transform.TransformException: com.android.ide.common.pro
cess.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.
DexException: Multiple dex files define Lcom/google/android/gms/internal/zzev;

But then I replace this code:
cordova.system.library.8=com.google.android.gms:play-services-base:11.0.4
cordova.system.library.9=com.google.android.gms:play-services-ads:11.0.4

in project.properties

With this:
cordova.system.library.8=com.google.android.gms:play-services-base:11.+
cordova.system.library.9=com.google.android.gms:play-services-ads:11.+

And:
BUILD SUCCESSFUL

Dear @srikanth-wgl  Could you suggest me how to connect unity mediation (I need award video)?
I have links:
https://github.com/googleads/googleads-mobile-android-mediation/tree/master/ThirdPartyAdapters/unity

But I don't know steps how to build it. Could suggest any directions?

Thanks in advance.

**Comment by srikanth-wgl** [comment]
@ivanov84 As you already know, multiple dex files issue is being discussed in issue #187. You did not mention any 'package does not exist' error with admob-free@0.15.0. I am closing this issue. Fixed in PR #189.
